src/components/ViewOrder.jsx

```diff
@@ -5,31 +5,31 @@ import {Item, Container, Header, Button} from 'semantic-ui-react'
 class ViewOrder extends Component {
 
   state = {
-    menuData: [],
+    orderData: [],
```

Comment:
Why did you  change from menuData to orderData?

Reply:
because menuData didn't reflect the contents of variable